src/ansys/templates/python/osl_solution/{{cookiecutter.__project_name_slug}}/src/ansys/solutions/{{cookiecutter.__solution_name_slug}}/solution/problem_setup_step.py

```diff
@@ -29,8 +42,8 @@ class ProblemSetupStep(StepModel):
     project_locked: bool = False
 
     # Backend data model
-    tcp_server_host: Optional[str] = None
-    tcp_server_port: Optional[int] = None
+    osl_server_host: Optional[str] = None
```

Comment:
I don't think the solution needs the host and port info anymore as things are supposed to be handled by the instancemanager

Reply:
The monitoring pages in the client side requires this information.